pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/QueryRunner.java

```diff
@@ -244,10 +245,10 @@ public void processQuery(WorkerMetadata workerMetadata, StagePlan stagePlan, Map
 
       // Send error block to all the receivers if pipeline breaker fails
       if (pipelineBreakerResult != null && pipelineBreakerResult.getErrorBlock() != null) {
-        TransferableBlock errorBlock = pipelineBreakerResult.getErrorBlock();
+        ErrorMseBlock errorBlock = pipelineBreakerResult.getErrorBlock();
         int stageId = stageMetadata.getStageId();
         LOGGER.error("Error executing pipeline breaker for request: {}, stage: {}, sending error block: {}", requestId,
```

Comment:
Won't this just use default version of toString() method, producing useless message ?

Reply:
Yes and no. The older TransferableBlocks overrides toString. I've modifies MseBlocks to do the same.

Anyway, the error is pretty much the same, given the default toString includes the class name and that is mostly what we want to log.